caiman/source_extraction/cnmf/params.py

```diff
@@ -858,6 +861,7 @@ def __init__(self, fnames=None, dims=None, dxy=(1, 1),
         self.motion = {
             'border_nan': 'copy',               # flag for allowing NaN in the boundaries
             'gSig_filt': None,                  # size of kernel for high pass spatial filtering in 1p data
+            'interp_shifts_precisely': False,   # interpolate shifts based on patch locations instead of reizing
```

Comment:
I wonder if precise_shifts or something else that's a little more concise would be better.

Reply:
Definitely could change the name...I think that just "precise_shifts" might be too vague, since it's not about the values of the shifts themselves but just how they're interpolated?

Reply:
I'm not set on that name, but "interp" kind of bugs me and the whole thing seems a bit long.

Reply:
How about just "interpolate_shifts"? precisely is redundant, it's either interpolating or resizing

Reply:
Sure, that works.

Reply:
Oh, wait, that actually conflicts with the name of one of the functions I added. It's probably better as the name of a function. What if it was a string parameter instead, either "resize" or "interpolate"? Maybe call it "upsampling_method" or can you think of something better?

Reply:
Do we expect there to be other reasonable ways to do this we might want to implement someday? If not, a boolean should be fine.

If we want to make it string, upsample_method isn't bad, or maybe just upsample? shift_resize?

Reply:
> Do we expect there to be other reasonable ways to do this we might want to implement someday? If not, a boolean should be fine.

Yeah, I don't know, probably not. 

How about "pw_interpolate"?

Reply:
Heh, we're definitely proving that "naming things is hard" adage.

I just noticed the text says "reizing" rather than resizing - that needs a fix.

For the name, maybe shifts_interpolate is the most clear. "pw" feels unclear to me.

Reply:
Works for me!